README.md

```diff
@@ -27,6 +27,13 @@ function iterm2_print_user_vars() {
 }
 ```
 
+The above command can be slow if the internet connection is slow. An alternative is to use this function:
+```
+function iterm2_print_user_vars() {
+  awk '/^current-context:/{print $2;exit;}' <~/.kube/config
+}
+```
```

Comment:
when i try this, i just get my current context printed to the terminal, not set in iTerm's `kubecontext` var.

did you intend to do something like this:
```bash
function iterm2_print_user_vars() {
  iterm2_set_user_var kubecontext $(awk '/^current-context:/{print $2;exit;}' <~/.kube/config)
}
```

Reply:
Sorry, you're right :) I've committed an updated version with your change.